core/options/user-option.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Elementor\Core\Options;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+abstract class User_Option extends Option_Base {
+	public static $user_id;
+
+	public function __construct( $user_id ) {
+		static::$user_id = $user_id;
+	}
+
+	public static function get() {
+		$value = get_user_option( static::get_full_key(), static::$user_id );
+		if ( false === $value ) {
+			$value = static::get_default();
+		}
+		return $value;
```

Comment:
```suggestion
		$value = get_user_option( static::get_full_key(), static::$user_id );
		
		return ( false === $value ) ? static::get_default() : $value;
```

Or at least add some space:
```suggestion
		$value = get_user_option( static::get_full_key(), static::$user_id );
		
		if ( false === $value ) {
			$value = static::get_default();
		}
		
		return $value;
```